ecs/spotfleet-cluster/example/main.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_region" "current" {}
+
+module "vpc" {
+  source          = "../../../ec2/vpc"
+  prefix          = "${var.prefix}"
+  cidr_block      = "10.1.0.0/16"
+  tags            = "${var.tags}"
+  private_subnets = "0"
+}
+
+module "example_spotfleet-cluster" {
```

Comment:
Just call it `spotfleet`. To me, fleet and cluster mean the same thing.

Reply:
Also, I think it would be good to add some ECS services to the example (basically, just copy the ECS example). Even though we just run `terraform plan` for now, it's great if this can be the example we use when running `terraform apply` manually to test.

Reply:
Actually, you could consider moving the example under `ecs/example` and just calling it `spotfleet.tf`, then just duplicate the other resources so we can test both things at once?